### PR TITLE
Exportable Section 

### PR DESCRIPTION
**Pull request by Jmsmr**

### CODE REVIEW

File: app/controllers/admin/sections_controller.rb
Comment (eielelle) [inline app/controllers/admin/sections_controller.rb:1]: run rubocop

**Review by eielelle — CHANGES_REQUESTED** [review]
run rubocop

File: app/controllers/concerns/section_searchable.rb
Comment (eielelle) [inline app/controllers/concerns/section_searchable.rb:1]: use searchable concern instead of creating new one

**Review by eielelle — CHANGES_REQUESTED** [review]

**Comment by eielelle** [comment]
run rubocop

**Comment by Jmsmr** [comment]
> run rubocop
the error here still the bundle 
image

**Pull request merged**